File: src/main/java/kr/hahaha98757/lastweapons/events/TitleEvent.java
package kr.hahaha98757.lastweapons.events;

import net.minecraftforge.fml.common.eventhandler.Event;

public class TitleEvent extends Event {
    private final String title;

    public TitleEvent(String title) {
        this.title = title;
    }

    public String getTitle() {
        return this.title;
    }
}

File: src/main/kotlin/kr/hahaha98757/lastweapons/events/TitleEvent.kt
package kr.hahaha98757.lastweapons.events

import net.minecraft.util.EnumChatFormatting
import net.minecraftforge.fml.common.eventhandler.Event

class TitleEvent(private val title: String): Event() {
    fun getTitle(): String = EnumChatFormatting.getTextWithoutFormattingCodes(title)
}